### ISSUE TITLE
Precise docker install

### ISSUE THREAD

**Issue by fozak** [eduardoboucas/staticman#66]
Dear Eduardo and @zburgermeiszter first of all thank you very much for such a great product.  We would like to set up own staticman on DigitalOcean droplet and own domain. Unfortunately, we have no admin with Docker knowledge so, we would like to ask for very precise docker install guide (step by step) to make painless install (with all config nuances) . We did before docker installs with other software (discourse forum) but they have very much step by step guide - https://github.com/discourse/discourse/blob/master/docs/INSTALL-cloud.md. Do you have something similar or close to that instruction? 
Thanks
Dennis

**Comment by eduardoboucas** [comment]
@zburgermeiszter is this something you'd be able to help with? If so, perhaps it would make sense to keep it outside of README and move all Docker-related documentation to a separate file?

**Comment by zburgermeiszter** [comment]
@dennisd1 
On DigitalOcean you can install Docker from the admin panel when you create your droplet.
Then you only need to pull the `staticman` repo, configure it as you would normally do using the documentation (which is WIP unfortunately), and simply run `docker-compose up`.

But I'll help @eduardoboucas how to set up automated build on Dockerhub, so you don't need to build the Dockerfile locally.

I work on other features at the moment, but there'll be a Docker install howto for it.

**Comment by zburgermeiszter** [comment]
@eduardoboucas 
Please set up an [automated Docker Hub](https://docs.docker.com/docker-hub/github/) build from this repo. It's quite simple, but require your permissions to set up a webhook to automatically rebuild the image every time there's a change in the git `master` branch.

For now, don't worry about the labels if you are not familiar with Docker Hub, simply link the repo to a Docker Hub build.

It is suggested to set up the same vendor/repo relation as on GitHub. So your username should be `eduardoboucas` and the image name is `staticman`

**Comment by fozak** [comment]
thanks a lot

**Comment by diogo-almeida** [comment]
@zburgermeiszter 

I've just installed staticman on my own server (a Digital Ocean droplet) via docker. While the installation seems to work (I configured the installation script by adding my GitHub token and private RSA key), I cannot access the point of connection for staticman. I've tried to point my browser to the correct url, even using the port number after the ip address (which is port 3000 by default, correct?), but I get no connection. 

For instance, let's say that my ip is 123.12.12.123 (completely made up). If I point my browser to "http://123.12.12.123:3000", I get an error message saying that the "server where this page is located is not responding". I get a similar message when I try inviting my staticman instance from my GitHub repo. When I go to https://123.12.12.123:3000/v2/connect/{my GitHub username}/{my repository name}, I get an error message, not the invitation that I sent.

I've been trying to solve this issue for a few days, but given that I am really a newbie when it comes to hosting apps on remote servers, I have had no success. Any obvious pointers regarding what I could do to solve this problem? Any help would be greatly appreciated.
